content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -1,215 +1,157 @@
 
-If you're already familiar with ORMs, feel free to jump to the [next section](#prisma) on Prisma.
+The idea with ORMs is that you define your models as **classes** that map to tables in a database. The classes and their instances provide you with a programmatic API to read and write data in the database.
+
+There are two common ORM patterns: [_Active Record_](https://en.wikipedia.org/wiki/Active_record_pattern) and [_Data Mapper_](https://en.wikipedia.org/wiki/Data_mapper_pattern) which differ in how they transfer data between objects and the database. While both patterns require you to define classes as the main building block. The most notable difference between the two is that the Data Mapper pattern decouples in-memory objects in the application code from the database and uses the data mapper layer to transfer data between the two. In practice, this means that with Data Mapper the in-memory objects (representing data in the database) don't even know that there’s a database present.
+
+#### Active Record
 
-### Concepts
+_Active Record_ ORMs map model classes to database tables where the structure of the two representations is closely related, e.g. each field in the model class will have a matching column in the database table. Instances of the model classes wrap database rows and carry both the data and the access logic to handle persisting changes in the database. Additionally, model classes can carry business logic specific to on the data in the model.
 
-ORMs provide a high-level database abstraction. They expose a programmatic interface to create, read, delete, and manipulate data while hiding some of the complexity of the database. Database abstractions vary in terms of the complexity they hide and the approach they take (for example, query builders like [knex.js](https://knexjs.org/) and [MassiveJS](https://massivejs.org/)).
+The model class typically has methods that do the following:
 
-The idea with ORMs is that you define your models as **classes** that map to tables in a database. The classes and their instances provide you with a flexible API to read and write data in the database.
+- Construct an instance of the model from an SQL query.
+- Construct a new instance for later insertion into the table.
+- Methods to wrap commonly used SQL queries and return Active Record objects.
+- Update the database and insert into it the data in the Active Record.
+- Get and set the fields.
+- Implement business logic.
 
-Database records are represented as instances of the model classes which carry the logic for storage, retrieval, serialization, and deserialization. They often also contain business logic.
+#### Data Mapper
 
-Model classes typically have several responsibilities:
+_Data Mapper_ ORMs, in contrast to Active Record, decouple the application's in-memory representation of data from the database's representation. The decoupling is achieved by requiring you to separate the mapping responsibility into two types of classes:
```

Comment:
Tiny thing but since it was introduced earlier, this felt a bit like repetition to what's just above. No big deal at all.